src/platforms/common/enriching-events/attachments/index.mdx

```diff
@@ -9,6 +9,7 @@ supported:
   - java
   - javascript
   - dotnet
+  - dart
```

Comment:
do we probably need to add `flutter` too?

Reply:
Shouldn't it show up for Flutter as is? Since Flutter basically extends Dart? Otherwise, I would just duplicate the Dart `includes` which doesn't seem right.

Reply:
Okay, it works like that. I must have been doing wrong before...